libp2p/kademlia/routed_host.py

```diff
@@ -0,0 +1,21 @@
+from libp2p.host.basic_host import BasicHost
+
+class RoutedHost(BasicHost):
+    def __init__(self, _network, _kad_network):
+        super(RoutedHost, self).__init__(_network)
+        self.kad_network = _kad_network
+
+    def get_kad_network(self):
+        return self.kad_network
+
+    def kad_listen(self, port, interface='0.0.0.0'):
```

Comment:
Should we call these functions routed_listen instead of kad_listen since kademlia is not the only way to do routing.

Reply:
Fixed!